server/api/adb.js

```diff
@@ -0,0 +1,12 @@
+import { getRecords } from "~~/components/airtable-demo"
+
+
+export default async (req, res) => {
+    try{
```

Comment:
Please activate prettier in VSCode. Code is not properly formatted.

Reply:
I have it activated